.project-inventory.json

```diff
@@ -0,0 +1,6 @@
+{
+  "classification": "cli-application",
+  "jiraKey": "ICOMP",
+  "jiraComponent": "",
+  "jiraSummaryPrefix": "PHP CodeSniffer Baseliner"
+}
```

Comment:
I prefer using JIRA key `IDEVT` and component `php`.

Reply:
Agreed, updated the file.